.gitignore

```diff
@@ -3,7 +3,6 @@ __pycache__/
 *.pyc
 *.sqlite3
 .vscode/
-media
```

Comment:
I see the point of adding the dockerignore for staging or production, but if we add it it will affect PRs since no media would be copied to the images.